tools/cldr-apps/js/src/esm/cldrClient.mjs

```diff
@@ -15,9 +15,9 @@ const RESOLVED_ROOT = new URL(OAS3_ROOT, document.baseURI);
  *
  * <https://github.com/swagger-api/swagger-js/blob/master/docs/usage/tags-interface.md#openapi-v3x>
  *
- * @returns Promise<SwaggerClient>
+ * @returns {Promise<SwaggerClient>}
  */
-function getClient() {
+async function getClient() {
   return new SwaggerClient({
```

Comment:
I thought I changed this code to cache the client. Maybe it's stuck in some PR.

Looking at https://github.com/swagger-api/swagger-js/blob/master/docs/migration/migration-2-x-to-3-x.md#promises 

I think the following is what we should be doing.

```suggestion
  return await SwaggerClient({
```

Reply:
I'll test that locally...

Reply:
the `async` nor `await` is not needed in that case actually. just pass `return SwaggerClient` as a Promise back to the callers who await it.

Reply:
I'd say if the PR as is is working let's address this in a future one?

Reply:
Oh, well... I just tested with changing new to await, and it worked, so I made a commit...

Reply:
I'll test `return SwaggerClient` and make that change if it works